src/main/java/com/sforce/cd/apexUnit/client/fileReader/ApexManifestFileReader.java

```diff
@@ -57,6 +57,11 @@
 public class ApexManifestFileReader {
 	private static Logger LOG = LoggerFactory.getLogger(ApexManifestFileReader.class);
 	public static ArrayList<String> nonExistantApexClassEntries = new ArrayList<String>();
+	private boolean includeTriggers;
+
+	public ApexManifestFileReader(boolean includeTriggers) {
```

Comment:
What is the behavior if new ApexManifestFileReader(); is used (by not passing parameters) ? Does it invoke QueryConstructor.generateQueryToFetchApexTrigger() method ?